blocks/cards/cards.css

```diff
@@ -203,6 +203,15 @@ main .cards.machine-parts ul > li {
   .cards > ul {
     flex-flow: row nowrap;
   }
+
+  .cards.columns-3 > ul {
```

Comment:
Why just for `columns-3` ? I'd say `cards` should always  `row wrap` in wide-screens and `columns` direction in small screens.

Reply:
@sdmcraft I don't know why this block supports `min-width: 48rem` and by default it's 
```
  .cards > ul {
    flex-flow: row nowrap;
  }
```
but I agree with you that will make the change.

Reply:
@jindaliiita , I think we added this via [this commit](https://github.com/hlxsites/sunstar/commit/55de3a40d8d0e63b90f54639d1dc8e79d3e8aaca) which imposes `row` direction of cards at `> 48rem` which seems to be different from what @JiangLong2019 has on his page. Let's discuss if there's a common solution possible.

Reply:
@sdmcraft @jindaliiita Done the change, please help to review.

Reply:
I'd need to do some adjustments in home page but that's fine.